CHANGES.md

```diff
@@ -45,9 +45,12 @@ To be released.
  -  Added `BlockHeader.TotalDifficulty` property.  [[#666], [#917]]
  -  The existing `Pong` message type (with the type number `0x02`) was
     replaced by a new `Pong` message type
-    (with the type number `0x14`).  [[#459]. [#919]]
+    (with the type number `0x14`).  [[#459], [#919], [#920], [#930]]
  -  The `TimestampThreshold` between `Block<T>`s was changed from 15 minutes to
     15 seconds.  [[#922], [#925]]
+ -  Added new message types:
+     -  Added `GetChainStatus` message.  [[#920], [#930]]
+     -  Added `ChainStatus` message.  [[#920], [#930]]
```

Comment:
```suggestion
     -  Added `GetChainStatus` (`0x20`) message.  [[#920], [#930]]
     -  Added `ChainStatus` (`0x21`) message.  [[#920], [#930]]
```

It's just suggestion, how about adding the message type value like the below lines in *CHANGES.md*, or as another format.

```
 -  `Swarm<T>` became to have two more message types: `GetRecentStates` (`0x0b`)
    and `RecentStates` (`0x0c`).  [[#272], [#343]]
```

https://github.com/planetarium/libplanet/blame/7723a3deb39ea49400e76862318ec482a2a179f9/CHANGES.md#L1074-L1075